pheps/phep-9999.md

```diff
@@ -0,0 +1,154 @@
+```
+PHEP: 9999
+Title: Community and Licensing Standards
+Author: Jonathan T. Niehof <jtniehof@gmail.com> <https://orcid.org/0000-0001-6286-5809>
+Discussions-To: https://github.com/heliophysicsPy/standards/pull/44
+Revision: 1
+Status: Draft
+Type: Standards Track
+Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+Requires: 4
+Created: 
```

Comment:
This is intentional trailing whitespace, which is why that precommit check fails. I have no idea why the "ligature characters with NFKD normalization" fails...

Reply:
Well, found a stray Unicode ligature and torched it (which means two pushes today, but just left the single entry for the date).